### PR TITLE
Update republish time and lightly reorganize republish info

### PR DESCRIPTION
**Pull request by neverett**
In this PR:

* Update to correct package republish time (24 hours)
* Light reorganization of republish info for clarity

### CODE REVIEW

File: doc/cli/npm-unpublish.md [hunk 1]
```diff
@@ -24,14 +24,13 @@ If no version is specified, or if all versions are removed then
 the root package entry is removed from the registry entirely.
 
 Even if a package version is unpublished, that specific name and
-version combination can never be reused.  In order to publish the
-package again, a new version number must be used.
+version combination can never be reused. In order to publish the
+package again, a new version number must be used. Additionally,
+new versions of unpublished packages may not be republished until 24 hours
```
Comment (ljharb) [inline doc/cli/npm-unpublish.md]: This doesn’t seem accurate; new versions can be published immediately, altho the old version can never be republished.

Is this a new requirement?

Comment (iarna) [reply doc/cli/npm-unpublish.md]: Entirely unpublished packages have a 24 hour delay before they can be published again. This isn't particularly new, but it's not widely known, thus these docs.

Comment (ljharb) [reply doc/cli/npm-unpublish.md]: ahhh ok so it’s only if there remains no published versions, then there’s a 24 hour delay?

Maybe the wording here could be like “fully unpublished” or “new versions of packages with every version unpublished”, so it’s clearer?

Comment (zkat) [reply doc/cli/npm-unpublish.md]: I like that rewording, tbh.

**Comment by neverett** [comment]
Thanks @ljharb! Updated with your wording.

**Review by ljharb — APPROVED** [review]
Much clearer to me, thanks <3

**Review by zkat — APPROVED** [review]
🎉 🐑 🚀 💥 

The only thing that'd be nice is if you wrapped the modified lines to the same length as their surrounding lines, but that's not a big deal if you don't get around to it.

Thank you!

**Pull request merged**